phys/module_sf_noahdrv.F

```diff
@@ -1416,6 +1423,10 @@ SUBROUTINE lsm(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
             if (I.eq.73.and.J.eq.125)THEN
                check = 1
             end if
+! Distributed aerodynamics
+            lf_urb_s = lf_urb2d_s(I, J)
+            z0_urb = z0_urb2d(I, J)
+            vegfrac = vegfra(I, J) / 100
```

Comment:
double check consistency for the vegfrac used in urban fraction (=1-vegfrac) and the original vegfra used in Noah.